artifacts.go

```diff
@@ -27,6 +26,9 @@ import (
 	"github.com/urfave/cli"
 )
 
+// VERSION of the mender-artifact CLI tool
+var VERSION = "0.1"
```

Comment:
@kacf this is quite isolated and can be handled separately: https://tracker.mender.io/browse/MEN-897